lib/unexpectedMitm.js

```diff
@@ -313,6 +313,20 @@ function determineInjectionCallsite(stack) {
     }
 }
 
+function responseByValidationRules(response, validationBlock) {
+    if (!validationBlock) {
+        return response;
+    }
+
+    if (validationBlock.ignoreHeaders) {
+        validationBlock.ignoreHeaders.forEach(function (headerKey) {
+            delete response.headers[headerKey.toLowerCase()];
```

Comment:
Actually the 'normal' conversation diffs happen first. This is the second round - what happens is we have the conversation, then we do the request to the real service ethics becomes the verification spec and then we 'to satisfy' between them. From what I remember (happy to look into it) these headers are removed from what will become the verification spec, meaning the LHS > RHS so the checks pass.